src/coreclr/vm/ilstubresolver.cpp

```diff
@@ -348,6 +348,43 @@ void ILStubResolver::SetLoaderHeap(PTR_LoaderHeap pLoaderHeap)
     m_loaderHeap = pLoaderHeap;
 }
 
+COR_ILMETHOD_DECODER* ILStubResolver::FinalizeILStub(ILStubLinker* sl)
+{
+    STANDARD_VM_CONTRACT;
+
+#ifndef DACCESS_COMPILE
```

Comment:
```suggestion
#ifndef DACCESS_COMPILE
COR_ILMETHOD_DECODER* ILStubResolver::FinalizeILStub(ILStubLinker* sl)
{
    STANDARD_VM_CONTRACT;

```
This method should not be needed in DAC build.